tasks/systemd.yml

```diff
@@ -6,3 +6,9 @@
     owner: "root"
     mode: "u=rwx,g=r,o=r"
   notify: "restart node_exporter"
+
+- name: "Make sure node exporter service is running and auto-starting"
+  service:
+    name: "node_exporter"
+    state: "started"
```

Comment:
How about `reloaded` instead of `started`? I mean, previous task is setting up service which means theoretically it might happen that it update its configuration so maybe reloading is better than just ensuring that it is running...